scripts/check_compliance.py

```diff
@@ -41,6 +41,10 @@ def git(*args):
     # Hack: Setting _cwd to the working dir seems pointless as of writing (it
     # should be the default), but keep it around in case it's working around
     # some issue
+
+    # pylint doesn't like how the sh library works.
+    #
+    # pylint: disable=unexpected-keyword-arg
```

Comment:
BTW "sh" is the only module that doesn't work on Windows AFAIR. Off-topic sorry.